src/org/labkey/test/components/ui/entities/EntityBulkInsertDialog.java

```diff
@@ -161,83 +166,93 @@ public String getDescription()
         return getWrapper().getFormElement(elementCache().description);
     }
 
-    public EntityBulkInsertDialog setTextField(String fieldKey, String value)
+    public EntityBulkInsertDialog setTextArea(CharSequence fieldIdentifier, String value)
     {
-        elementCache().textInput(fieldKey).set(value);
+        elementCache().textArea(fieldIdentifier).set(value);
         return this;
     }
 
-    public String getTextField(String fieldKey)
+    public String getTextArea(CharSequence fieldIdentifier)
     {
-        return elementCache().textInput(fieldKey).get();
+        return elementCache().textArea(fieldIdentifier).get();
     }
 
-    public EntityBulkInsertDialog setNumericField(String fieldKey, String value)
+    public EntityBulkInsertDialog setTextField(CharSequence fieldIdentifier, String value)
     {
-        elementCache().numericInput(fieldKey).set(value);
+        elementCache().textInput(fieldIdentifier).set(value);
         return this;
     }
 
-    public String getNumericField(String fieldKey)
+    public String getTextField(CharSequence fieldIdentifier)
     {
-        return elementCache().numericInput(fieldKey).get();
+        return elementCache().textInput(fieldIdentifier).get();
     }
 
-    public EntityBulkInsertDialog setSelectionField(String fieldCaption, List<String> selectValues)
+    public EntityBulkInsertDialog setNumericField(CharSequence fieldIdentifier, String value)
     {
-        FilteringReactSelect reactSelect = elementCache().selectionField(fieldCaption);
+        elementCache().numericInput(fieldIdentifier).set(value);
+        return this;
+    }
+
+    public String getNumericField(CharSequence fieldIdentifier)
+    {
+        return elementCache().numericInput(fieldIdentifier).get();
+    }
+
+    public EntityBulkInsertDialog setSelectionField(CharSequence fieldIdentifier, List<String> selectValues)
+    {
+        FilteringReactSelect reactSelect = elementCache().selectionField(fieldIdentifier);
         selectValues.forEach(reactSelect::filterSelect);
         return this;
     }
 
-    public List<String> getSelectionField(String fieldCaption)
+    public List<String> getSelectionField(CharSequence fieldIdentifier)
     {
-        FilteringReactSelect reactSelect = elementCache().selectionField(fieldCaption);
+        FilteringReactSelect reactSelect = elementCache().selectionField(fieldIdentifier);
         return reactSelect.getSelections();
     }
 
     /**
      * Clear the value(s) from a field that is a drop down selection field.
      *
-     * @param fieldCaption Caption for the field.
+     * @param fieldIdentifier Name for the field.
```

Comment:
```suggestion
     * @param fieldIdentifier Identifier (name or fieldKey) for the field.
```
Would this be more accurate?